operations/host_spawn.go

```diff
@@ -480,6 +480,58 @@ func hostStart() cli.Command {
 	}
 }
 
+func hostSSH() cli.Command {
+	const (
+		identityFlagName = "identity"
+	)
+
+	return cli.Command{
+		Name:  "ssh",
+		Usage: "ssh into a spawn host",
+		Flags: addHostFlag(
+			cli.StringFlag{
+				Name:  joinFlagNames(identityFlagName, "i"),
+				Usage: "Path to a specific identity (private key), for ssh -i",
+			},
+		),
+		Before: mergeBeforeFuncs(setPlainLogger, requireHostFlag),
+		Action: func(c *cli.Context) error {
+			confPath := c.Parent().Parent().String(confFlagName)
+			hostID := c.String(hostFlagName)
+			key := c.String(identityFlagName)
+
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
+
+			conf, err := NewClientSettings(confPath)
+			if err != nil {
+				return errors.Wrap(err, "problem loading configuration")
+			}
+			client := conf.setupRestCommunicator(ctx)
+			defer client.Close()
+
+			h, err := client.GetSpawnHost(ctx, hostID)
```

Comment:
Since we allow people to name hosts, should we also look for the host by name? Also, if someone only has one workstation, could we return that one?

Reply:
we don't (yet) validate the hosts' names are unique

Reply:
I also don't want to just return the one workstation, since this code isn't technically workstation specific. But ids don't change so users could easily make an alias for their workstation.